src/components/form/Editor.tsx

```diff
@@ -73,67 +73,63 @@ export const FormItemEditor = <T extends FieldValues>(
   const { t } = useTranslation();
   const { controllerProps, restProps } = genControllerProps(props, '');
   const { customSchema, language, isLoading, ...wrapperProps } = restProps;
-  const { setError, clearErrors } = useFormContext<{
-    [name: string]: object;
-  }>();
-  const customErrorField = `${props.name}-editor`;
-  const { errors } = useFormState({ name: customErrorField });
+  const { trigger } = useFormContext();
+  const monacoErrorRef = useRef<string | null>(null);
```

Comment:
The original method was to directly use `setError` and `clearError`, which would trigger a global update of the form with every input. Also, due to the various dependencies on state, it might update multiple times.

Now, using `ref` allows React and RHF to ignore state updates, and only when `onValidate` is triggered, actively call `trigger`, which enables RHF to obtain an error (ref) once through internal validation logic. This significantly reduces the dependencies on state and limits error updates within the component.